frontera/contrib/messagebus/zeromq/socket_config.py

```diff
@@ -1,29 +1,62 @@
 # -*- coding: utf-8 -*-
+"""
+Contains the SocketConfig class
+"""
+from socket import getaddrinfo, gaierror
 
 
 class SocketConfig(object):
-
-    hostname = None
-    base_port = None
-
-    def __init__(self, hostname, base_port):
-        self.hostname = hostname
-        self.base_port = base_port
+    """
+    Converts address to IPv4 or IPv6 or * and returns the necessary socket
+    addresses.
+    NOTE: When using * it defaults to IPv4
```

Comment:
Actually by using different ports for the broker, IPv6 testing should now be possible [see here](https://github.com/scrapinghub/frontera/pull/108/files#diff-d8026c0965fad80de128c149837278f9R81) (also the wildcard test could be added there).
I also updated some docs with my newest commit.
I do not think that I have the time tomorrow to make testing of IPv6 and wildcard work on Travis. But even if Travis testing worked it would only test this on `ip6-localhost`.. not in a distributed setup. So I could not say for sure if IPv6 would work.
Of course after friday you can happily take over. In my opinion IPv6 support could still be in a separate pull request since my changes will not break anything.